components/Testimonies.js

```diff
@@ -0,0 +1,57 @@
+import React, { useRef, useState } from 'react';
+import Image from 'next/image';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faScrewdriverWrench } from '@fortawesome/free-solid-svg-icons'
+import { faWindows, faApple } from '@fortawesome/free-brands-svg-icons';
+import styles from './Testimonies.module.css';
+
+export default function Testimonies() {
+    const TestimonyData = [
```

Comment:
Please use `lowerCamelCase` for variables as per https://google.github.io/styleguide/tsguide.html#naming-rules-by-identifier-type